extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLog.i18n.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+$messages = array();
+
+$messages['en'] = array(
```

Comment:
There's no Crowdin conf file as i18n is not a requirement for prod, but all the strings are here to make it easier in the future.